ReadIt/ReadingList/ReadingCell.swift

```diff
@@ -7,17 +7,139 @@ import UIKit
 		super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
 
-		self.textLabel?.font = UIFont(name: "Palatino", size: 20)
+		setupViewConfiguration()
+	}
+	
+	private func setupViewConfiguration() {
+		buildViewHierarchy()
+		setupConstraints()
+	}
+	
+	private func buildViewHierarchy() {
+		contentView.addSubview(banner)
+		contentView.addSubview(titleLabel)
+		contentView.addSubview(authorLabel)
+		contentView.addSubview(descriptionLabel)
+		contentView.addSubview(estimatedTimeLabel)
+	}
+	
+	private func setupConstraints() {
+		banner.centerYAnchor.constraint(equalTo: contentView.centerYAnchor).isActive = true
+		banner.topAnchor.constraint(greaterThanOrEqualTo: contentView.topAnchor, constant: 8).isActive = true
+		banner.bottomAnchor.constraint(lessThanOrEqualTo: contentView.bottomAnchor, constant: -8).isActive = true
+		banner.trailingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.trailingAnchor).isActive = true
+		banner.widthAnchor.constraint(equalToConstant: 80).isActive = true
+
+		let bannerHeight = banner.heightAnchor.constraint(equalToConstant: 80)
+		bannerHeight.priority = .defaultHigh
+		bannerHeight.isActive = true
+		
+		titleLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16).isActive = true
+		titleLabel.leadingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leadingAnchor).isActive = true
+		titleLabel.trailingAnchor.constraint(equalTo: banner.leadingAnchor, constant: -8).isActive = true
+
+		authorLabelConstraints = [
+			authorLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 6),
+			authorLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
+			authorLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor),
+			descriptionLabel.topAnchor.constraint(equalTo: authorLabel.bottomAnchor, constant: 8)
+		]
+		
+		let authorLabelSkippingConstraint = descriptionLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 8)
+		authorLabelSkippingConstraint.priority = UILayoutPriority(201)
+		authorLabelSkippingConstraint.isActive = true
 
-		self.detailTextLabel?.font = .systemFont(ofSize: 11)
-		self.detailTextLabel?.textColor = .gray
+		descriptionLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor).isActive = true
+		descriptionLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor).isActive = true
+		
+		estimatedTimeLabel.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: 6).isActive = true
+		estimatedTimeLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor).isActive = true
+		estimatedTimeLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor).isActive = true
+		estimatedTimeLabel.bottomAnchor.constraint(lessThanOrEqualTo: contentView.bottomAnchor, constant: -8)
+
+		let bottomBreakableConstraint = estimatedTimeLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8)
+		bottomBreakableConstraint.priority = UILayoutPriority(200)
```

Comment:
This is `200` cause it needs to be less than `UILayoutPriority.defaultLow` (which is 250) otherwise, it may shrink the content instead